commands/utils/utils.go

```diff
@@ -23,3 +28,25 @@ func Chdir(dir string) (cbk func(), err error) {
 	}
 	return func() { err = os.Chdir(wd) }, err
 }
+
+func ReportUsage(commandName string, serverDetails *config.ServerDetails, usageReportSent chan<- error) {
+	var err error
+	defer func() {
+		// Signal usage report finished
+		usageReportSent <- err
+	}()
+	if serverDetails.ArtifactoryUrl == "" {
+		return
+	}
+	clientLog.Debug(usage.ReportUsagePrefix + "Sending info...")
+	serviceManager, err := utils.CreateServiceManager(serverDetails, -1, 0, false)
+	if err != nil {
+		clientLog.Debug(usage.ReportUsagePrefix + err.Error())
+		return
+	}
+	err = usage.SendReportUsage(productId, commandName, serviceManager)
+	if err != nil {
+		clientLog.Debug(err.Error())
```

Comment:
Let'a addd the ```usage.ReportUsagePrefix``` prefix, as the logging above this one.

Reply:
SendReportUsage in the client already adds this prefix